src/SB/Game/zEntSimpleObj.cpp

```diff
@@ -2,21 +2,727 @@
+                if (var_r25 > 8U)
+                {
+                    var_r3 = 0;
+                    temp_r0_2 = entCount - 1;
+                    var_ctr = (u32)(temp_r6_3 + 7) >> 3U;
+                    if (temp_r6_3 > 0U)
+                    {
+                        do
+                        {
+                            *(sSimpleCustomList + var_r3) =
+                                *(temp_r27 + ((temp_r0_2 - var_r4) * 4));
+                            *(sSimpleCustomList + (var_r3 + 4)) =
+                                *(temp_r27 + ((temp_r0_2 - (var_r4 + 1)) * 4));
+                            *(sSimpleCustomList + (var_r3 + 8)) =
+                                *(temp_r27 + ((temp_r0_2 - (var_r4 + 2)) * 4));
+                            temp_r10 = (temp_r0_2 - (var_r4 + 5)) * 4;
+                            temp_r8 = (temp_r0_2 - (var_r4 + 6)) * 4;
+                            temp_r11 = var_r3 + 0x10;
+                            temp_r9 = var_r3 + 0x14;
+                            *(sSimpleCustomList + (var_r3 + 0xC)) =
+                                *(temp_r27 + ((temp_r0_2 - (var_r4 + 3)) * 4));
+                            temp_r7 = var_r3 + 0x18;
+                            temp_r5 = var_r3 + 0x1C;
+                            temp_r6_4 = (temp_r0_2 - (var_r4 + 7)) * 4;
+                            temp_r28 = *(temp_r27 + ((temp_r0_2 - (var_r4 + 4)) * 4));
+                            var_r3 += 0x20;
+                            var_r4 += 8;
+                            *(sSimpleCustomList + temp_r11) = temp_r28;
+                            *(sSimpleCustomList + temp_r9) = *(temp_r27 + temp_r10);
+                            *(sSimpleCustomList + temp_r7) = *(temp_r27 + temp_r8);
+                            *(sSimpleCustomList + temp_r5) = *(temp_r27 + temp_r6_4);
+                            var_ctr -= 1;
+                        } while (var_ctr != 0);
+                    }
+                }
+                var_r6 = var_r4 * 4;
+                var_ctr_2 = var_r25 - var_r4;
+                if (var_r4 < var_r25)
+                {
+                    do
+                    {
+                        temp_r0_3 = ((entCount - 1) - var_r4) * 4;
+                        var_r4 += 1;
+                        *(sSimpleCustomList + var_r6) = *(temp_r27 + temp_r0_3);
+                        var_r6 += 4;
+                        var_ctr_2 -= 1;
+                    } while (var_ctr_2 != 0);
+                }
+            }
+        }
+        else
+        {
+            sSimpleCustomCount = 0;
+            sSimpleCustomList = NULL;
+        }
+        if (var_r26 == 0)
+        {
+            RwFree(temp_r27);
+            return;
+        }
+        sMgrCount = var_r26;
+        sMgrList = (xSphere*)xMemAlloc(gActiveHeap, var_r26 << 6, 0x40);
+        var_r29_2 = temp_r27;
+        var_r30_2 = 0U;
+        var_r25_2 = sMgrList;
+        // while (var_r30_2 < var_r26)
```

Comment:
What's all this code? Is this function WIP? No change needed if that's the case but I just want to make sure this isn't something you forgot to delete.